esm/interface/css-style-declaration.js

```diff
@@ -13,11 +13,10 @@ const updateKeys = style => {
     if (attr) {
       style.set(PRIVATE, attr);
       for (const rule of attr[VALUE].split(/\s*;\s*/)) {
-        const pair = rule.split(/\s*:\s*/);
-        if (1 < pair.length) {
-          let [key, value] = pair;
+        let [key, ...value] = rule.split(':');
+        if (value.length > 0) {
           key = key.trim();
-          value = value.trim();
+          value = value.join(':').trim();
```

Comment:
not sure why I've asked to make `key` mutable and we ended up with `value` changing type at runtime, which is never desired perf-wise-speaking 🤔

please rollback this last change and just make `const` a `let` or put back the `rest` reference without changing references type at runtime, thanks!

Reply:
My bad, sorry — not too familiar with these performance considerations. Thanks for the very thorough reviews 🙏